types/object-diff/index.d.ts

```diff
@@ -3,5 +3,5 @@
 // Definitions by: Rogier Schouten <https://github.com/rogierschouten>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-declare function diff(a: any, b: any): any;
+declare function diff(a: any, b: any, c?: any): any;
```

Comment:
I think instead of adding a third parameter, we should add in multiple optional arguments, like this:
```ts
declare function diff(a: any, ...args: any[]): any;
```
since the original JS function only expects us to pass 1 or more objects to diff. This would not only support a third parameter but any number of additional parameters.

Reply:
That makes sense to me!